NativeApp/src/components/User/View/index.js

```diff
@@ -44,6 +42,17 @@ const UserView = (props) => {
                   duration={item.halfDay ? 0.5 : getDuration(item.start, item.end)}
                 />
               )}
+              renderSectionHeader={({ section }) => (
+                section.data.length >= 1
+                  ? (<H3 type="bold" style={styles.sectionListHeader}>{section.title}</H3>)
+                  : (
+                    <View>
+                      <H3 type="bold" style={styles.sectionListHeader}>{section.title}</H3>
+                      <H3 style={styles.sectionListNoItems}>Nothing to show</H3>
+                    </View>)
```

Comment:
Is there anywhere else you can display the "nothing to show"? Feels weird to be sticking a div and 2 h3's inside 1 header.

Reply:
I kinda think that multiline ternaries like these are a bit untidy. Another approach would be to move the logic to a function that will return either component, or ideally do this kind of logic in a container that will pass the correct component through on props.

Reply:
i have made changes as per you suggestions. please could you check it out? thanks :)